### ISSUE TITLE
Console logging on every run

### ISSUE THREAD

**Issue by jonnyzzz** [jk1/TeamCity-dependencies-gradle-plugin#21]
I see the following line on every gradle build start

```
:: loading settings :: url = jar:file:/Users/jonnyzzz/.gradle/wrapper/dists/gradle-3.3-all/6kxd5x4zmx1m7xl2r691ucw2z/gradle-3.3/lib/plugins/ivy-2.2.0.jar!/org/apache/ivy/core/settings/ivysettings.xml

```

**Comment by jk1** [comment]
That's ivy implementation to blame here: http://grepcode.com/file/repo1.maven.org/maven2/org.apache.ivy/ivy/2.4.0/org/apache/ivy/core/settings/IvySettings.java

**Comment by jk1** [comment]
The fix is to be included into v1.0
